### ISSUE TITLE
Selecting "watch filesystem for changes" doesn't actually enable the option

### ISSUE THREAD

**Issue by TiZ-EX1** [syncthing/syncthing-gtk#530]
Syncthing has had a built-in filesystem watcher for some time now. You can enable a folder to be watched in the web UI, and it will set rescan time to one hour and then start watching changes. You can change a file and it will immediately sync. If you tick the corresponding box in the Syncthing-GTK UI without bothering to check the web UI, and then change a file, it won't sync. You'll have to wait up to an hour for the change to propagate! It seems that the checkbox here doesn't affect the underlying Syncthing setting.

I know Syncthing-GTK used to include a file watcher. Maybe it still does? Maybe there's some sort of thing going on where it knows that it's on a Syncthing version with the built-in watcher and doesn't have to use its own watcher, but fails to actually correctly set the configuration, and thus nobody is actually watching the folder at all. I don't know, just spitballing here.

Thanks for your work on this app.

**Comment by kozec** [comment]
> I know Syncthing-GTK used to include a file watcher. Maybe it still does?

No, that setting should reflect directly to syncthing config now, in same way as enabling watcher from webui does. I'll check what could be wrong.
